src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20200904133558_EES-1344-PreviousVersionIdShouldAllowNull.cs

```diff
@@ -0,0 +1,30 @@
+﻿using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Microsoft.EntityFrameworkCore.Migrations;
+using static GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations.MigrationConstants;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    public partial class EES1344PreviousVersionIdShouldAllowNull : Migration
+    {
+        private const string MigrationId = "20200904133558";
+        private const string PreviousVersionMigrationId = "20200713151302";
```

Comment:
This is the previous version that ReleaseType was updated in, but not the DropAndCreateRelease procedure